README.md

```diff
@@ -46,27 +46,20 @@ Tests written using mocha + schedule.js.
 
 ## Documentation
 
-- [Install](http://liskhq.github.io/lisk-js/index.html)
-- [API](http://liskhq.github.io/lisk-js/example/api.html)
-	- [Settings](http://liskhq.github.io/lisk-js/example/api.html#settings)
-	- [API Functions](http://liskhq.github.io/lisk-js/example/api.html#api_functions)
-	- [Network Functions](http://liskhq.github.io/lisk-js/example/api.html#network_functions)
-- [Crypto](http://liskhq.github.io/lisk-js/example/api.html#crypto)
-- [Transactions](http://liskhq.github.io/lisk-js/example/api.html#transactions)
-	- [Create Transaction](http://liskhq.github.io/lisk-js/example/api.html#functions_createTransaction)
-	- [Create Vote](http://liskhq.github.io/lisk-js/example/api.html#functions_createVote)
-	- [Create Dapp](http://liskhq.github.io/lisk-js/example/api.html#functions_createDapp)
-	- [Create Delegate](http://liskhq.github.io/lisk-js/example/api.html#functions_createDelegate)
-	- [Create Second Signature](http://liskhq.github.io/lisk-js/example/api.html#functions_createSignature)
-	- [Create Multisignature Account](http://liskhq.github.io/lisk-js/example/api.html#functions_createMultisignature)
-	- [Sign Multisignature Transaction](http://liskhq.github.io/lisk-js/example/api.html#functions_signMultisignature)
-- [Experiment (live)](http://liskhq.github.io/lisk-js/example/experiment.html)
-	- [Get Account Info](http://liskhq.github.io/lisk-js/example/experiment.html#get_account)
-	- [Send LSK](http://liskhq.github.io/lisk-js/example/experiment.html#send_lsk)
-	- [Sign Message](http://liskhq.github.io/lisk-js/example/experiment.html#sign)
-	- [Verify Message](http://liskhq.github.io/lisk-js/example/experiment.html#verify)
-	- [Encrypt Message](http://liskhq.github.io/lisk-js/example/experiment.html#encrypt)
-	- [Decrypt Message](http://liskhq.github.io/lisk-js/example/experiment.html#decrypt)
+- [Install](https://docs.lisk.io/docs/lisk-js-installation)
+- [API](https://docs.lisk.io/docs/api-functions)
+	- [Settings](https://docs.lisk.io/docs/api)
+	- [API Functions](https://docs.lisk.io/docs/api-functions)
+	- [Network Functions](https://docs.lisk.io/docs/network-functions)
+- [Crypto](https://docs.lisk.io/docs/crypto-functions)
+- [Transactions](https://docs.lisk.io/docs/transactions-1)
+	- [Create Transaction](https://docs.lisk.io/docs/transactions-1#section-createtransaction)
+	- [Create Vote](https://docs.lisk.io/docs/transactions-1#section-createvote)
+	- [Create Dapp](https://docs.lisk.io/docs/transactions-1#section-createdapp)
+	- [Create Delegate](https://docs.lisk.io/docs/transactions-1#section-createdelegate)
+	- [Create Second Signature](https://docs.lisk.io/docs/transactions-1#section-createtransaction)
+	- [Create Multisignature Account](https://docs.lisk.io/docs/transactions-1#section-createmultisignature)
+	- [Sign Multisignature Transaction](https://docs.lisk.io/docs/transactions-1#section-signtransaction)
```

Comment:
We should match the readme.io structure exactly

Reply:
It will be updated later when documentation merges to 1.0